src/s2/value_lexicon.h

```diff
@@ -28,14 +28,6 @@
 #include "s2/base/integral_types.h"
 #include "s2/util/gtl/dense_hash_set.h"
 
-class S2Point;
-namespace absl {
-namespace hash_internal {
-template <typename T>
-struct Hash;
```

Comment:
I'm sure this was added because something was failing, but it's obviously wrong and we shouldn't be depending on abseil internals.  I guess we can play whack-a-bug.